Project/src/main/java/app/FileUploadController.java

```diff
@@ -74,4 +71,15 @@ public String uploadFile(@RequestParam("file") MultipartFile file) throws IOExce
         return response;
     }
 
+    /**
+     * Deletes an article from the article repository
+     * @param id The id of the article to be deleted
+     * @return
+     */
+    @DeleteMapping("/upload/{id}")
+    public ResponseEntity<Void> deleteFileById(@PathVariable(value = "id") Long id) {
+        if (id <= 0 || !articleRepository.existsById(id)) throw new BadFileException();
```

Comment:
Isnt id <=0 useless check? articleRepository.existsById(id) should handle zero and negative numbers as well